dbt_subprojects/daily_spellbook/models/_projects/gmx/event/arbitrum/gmx_v2_arbitrum_event_schema.yml

```
@@ -1336,3 +1314,884 @@ models:
+        description: The address that initiated the transaction.
+      - name: tx_to
+        description: The destination address of the transaction.
+      - name: event_name
+        description: The type of event recorded, always `GLVCreated` for this model.
+      - *msg_sender
+      - name: glv_token
+        description: The identifier of the GLV token created in the transaction.
+      - name: long_token
+        description: The long token associated with the GLV creation.
+      - name: short_token
+        description: The short token associated with the GLV creation.
+      - name: salt
+        description: A unique salt value used in the GLV creation.
+      - name: glv_type
+        description: The type of GLV created.
+      - name: market_token_symbol
+        description: The symbol of the market token involved in the GLV creation.
+      - name: market_token_decimals
+        description: The decimal precision of the market token involved in the GLV creation.
+
+
+  - name: gmx_v2_arbitrum_glv_deposit_cancelled
+    meta:
+      blockchain: arbitrum
+      sector: dex
+      project: gmx
+      contributors: ai_data_master, gmx-io
+    config:
+      tags: ['arbitrum', 'gmx', 'event', 'glv_deposit_cancelled']
+    description: |
+      Processes decoded event data for `GLVDepositCancelled` events on the GMX platform in the Arbitrum blockchain.
+      This model extracts key details such as the account address, cancellation reason, and transaction specifics, enabling
+      analysis of cancelled GLV deposits.
+
+    columns:
+      - *blockchain
+      - *block_time
+      - *block_date
+      - *block_number
+      - *tx_hash
+      - *index
+      - *contract_address
+      - name: tx_from
+        description: The address that initiated the transaction.
+      - name: tx_to
+        description: The destination address of the transaction.
+      - name: event_name
+        description: The type of event recorded, always `GLVDepositCancelled` for this model.
+      - *msg_sender
+      - name: account
+        description: The account address associated with the cancelled deposit.
+      - name: key
+        description: A unique identifier for the cancelled deposit.
+      - name: reason_bytes
+        description: Encoded reason data for the deposit cancellation.
+      - name: reason
+        description: The decoded textual reason for the deposit cancellation.
+
+
+  - name: gmx_v2_arbitrum_glv_deposit_created
+    meta:
+      blockchain: arbitrum
+      sector: dex
+      project: gmx
+      contributors: ai_data_master, gmx-io
+    config:
+      tags: ['arbitrum', 'gmx', 'event', 'glv_deposit_created']
+    description: |
+      Processes decoded event data for `GLVDepositCreated` events on the GMX platform in the Arbitrum blockchain.
+      This model extracts key details such as the account address, deposit details, and transaction specifics, enabling
+      analysis of created GLV deposits.
+
```

We need to apply uniqueness tests to every incremental spell.
```suggestion
    data_tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - tx_hash
            - index
```

---

Thank you for reminder. I will update it everywhere :)

---

Thank you for reminder. I will update it everywhere :)